src/core/api.c

```diff
@@ -1083,8 +1083,11 @@ MsQuicStreamSend(
     SendRequest->ClientContext = ClientSendContext;
 
     SendInline =
-        !Connection->Settings.SendBufferingEnabled &&
-        Connection->WorkerThreadID == CxPlatCurThreadID();
+        !Connection->Settings.SendBufferingEnabled
+#ifdef CXPLAT_AT_DISPATCH
```

Comment:
why not have a dummy macro for other platforms?

Reply:
I can, but this required fewer files to be changed. That's why I did this instead.

Reply:
I think only 2 headers needed to be modified. We should be consistent with what we did for CXPLAT_PASSIVE_CODE and CXPLAT_IRQL.

Reply:
Fixed